functions/controllers/application/index.js

```diff
@@ -127,4 +127,14 @@ application.post("/submit", jwtCheck, hasUpdateApp, async (req, res) => {
   }
 });
 
+application.get("/checkApp", jwtCheck, hasReadApp, async (req, res) => {
+  try {
+    doc = await queryDocument("applicants", req.user.sub);
+    appStatus = doc.get("status");
+    res.status(200).send({ code: 200, status: appStatus, exists: true });
+  } catch (error) {
```

Comment:
Realized this while doing your documentation but can we get a message field for consistency.

Reply:
I was also wondering at line 134, we are returning a promise for applicationstatus: 
` appStatus = doc.get("status");`
`status: appStatus`.
Do we want to do this instead of grabbing the status field from the document snapshot and returning a string?
Or am I confused and `appStatus` will resolve to the field value when calling this endpoint?

Reply:
I believe doc.get uses the document snapshot so it shouldn't be a promsie, but please tell me if I'm wrong. For the message I'll do that really quick